README.md

```diff
@@ -102,13 +81,6 @@ High level `make` targets:
 * `system-test`: run the networking/"sanity" tests. Specify
   `CONTIV_NODE_OS=centos` to test on centos instead of ubuntu.
 
-#### Resource Allocation
```

Comment:
I think if we put it at the root, github will link to it when submitting PRs and issues.